releasing/helpers.sh

```diff
@@ -24,7 +24,7 @@ function refreshMaster {
 }
 
 function testKustomizeRepo {
-  make prow-presubmit-check >& /tmp/k.txt
+  make IS_LOCAL=true verify-kustomize-repo >& /tmp/k.txt
```

Comment:
@natasha41575 I know we discussed potentially including both, but after looking at their definitions, I believe that all `prow-presubmit-check`s are included in `verify-kustomize-repo`.